spek-api/src/main/kotlin/org/jetbrains/spek/api/IgnoreSpek.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.spek.api
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+annotation class IgnoreSpek
```

Comment:
New runtime discovers anonymous non-abstract inner classes, I want to ignore them. Also a fix for #200.